draft-ietf-quic-recovery.md

```diff
@@ -1562,6 +1569,7 @@ window.
        congestion_window *= kLossReductionFactor
        congestion_window = max(congestion_window, kMinimumWindow)
        ssthresh = congestion_window
+       bytes_acked *= kLossReductionFactor
```

Comment:
move this up to the line below where the congestion window is reduced

Reply:
Done. Thanks.